### PR TITLE
Reconstructed Alert frontend

### PR DESCRIPTION
**Pull request by hannahpurcell**
**Asana task**: [Reconstructed Alert Widget front-end](https://app.asana.com/0/1185117109217413/1201752179318950)

Reconstructed alerts built. Known issues:
- Route Pill on takeover alerts has wrong font size
- Alert language needs to be updated in places

- [ ] Needs version bump?

### CODE REVIEW

File: assets/css/v2/pre_fare/reconstructed_alert.scss [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+    }
+    
+    .route-pill__text {
+      position: relative;
+      width: fit-content;
+      padding: 0 40px;
+      font-size: 60px;
+      line-height: 80px;
+      text-transform: uppercase;
+      font-family: HelveticaNeue-Bold;
```
Comment (cmaddox5) [inline assets/css/v2/pre_fare/reconstructed_alert.scss]: My browser is having trouble loading this font. I changed it to `font-family: "neue-haas-grotesk-text";` and that makes it look like the design. Are you having the same issue or is it just me?

**Review by cmaddox5 — CHANGES_REQUESTED** [review]
Looks great! Just have a few suggestions plus some small bugs need fixing, but we are already talking about those in Slack.

File: lib/screens/v2/widget_instance/serializer/route_pill.ex [hunk 2]
```diff
@@ -217,6 +218,7 @@ defmodule Screens.V2.WidgetInstance.Serializer.RoutePill do
   defp get_color_for_route("Mattapan", _), do: :red
   defp get_color_for_route("Orange", _), do: :orange
   defp get_color_for_route("Green-" <> _, _), do: :green
+  defp get_color_for_route("Green" <> _, _), do: :green
```
[the remark this answers is not in the document]
Comment (hannahpurcell) [reply lib/screens/v2/widget_instance/serializer/route_pill.ex:221]: Lol, true! Silly I missed that :joy:

File: lib/screens/v2/widget_instance/reconstructed_alert.ex [hunk 3]
```diff
@@ -108,7 +128,14 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
       text:
         ["No"] ++
           (affected_routes
-           |> Enum.map(fn route -> %{icon: route} end)
+           |> Enum.map(fn route ->
+             %{
+               route:
+                 route
+                 |> String.replace("-", "_")
+                 |> String.downcase()
```
[the remark this answers is not in the document]
Comment (hannahpurcell) [reply lib/screens/v2/widget_instance/reconstructed_alert.ex:136]: Yeah, the frontend expects strings in the format "color_a", as in "orange" or "green_e"

File: assets/css/v2/pre_fare/reconstructed_alert.scss (hunk 1 above)
Comment (hannahpurcell) [reply assets/css/v2/pre_fare/reconstructed_alert.scss]: Yep, able to reproduce in FF. Thanks!

File: test/screens/v2/candidate_generator/widgets/reconstructed_alert_test.exs [hunk 4]
```diff
@@ -78,6 +78,15 @@ defmodule Screens.V2.CandidateGenerator.Widgets.ReconstructedAlertTest do
         ["place-hsmnl", "place-bckhl", "place-rvrwy", "place-mispk"]
       ]
 
+      stop_sequences = [
+        ["70260", "70258", "70256", "70254"],
+        ["70253", "70255", "70257", "70260"]
+      ]
+
+      station_sequences = [
```
[the remark this answers is not in the document]
Comment (hannahpurcell) [reply test/screens/v2/candidate_generator/widgets/reconstructed_alert_test.exs:86]: whoops, merge issue

**Review by cmaddox5 — APPROVED** [review]
Looks great!

File: config/dev.exs [hunk 5]
```diff
@@ -22,7 +22,7 @@ config :screens, ScreensWeb.Endpoint,
   ]
 
 config :screens,
-  default_api_v3_url: System.get_env("API_V3_URL", "https://api-v3.mbta.com/"),
+  default_api_v3_url: System.get_env("API_V3_URL", "https://api-dev-green.mbtace.com/"),
```
[the remark this answers is not in the document]
Comment (hannahpurcell) [reply config/dev.exs:25]: Whoops!! Definitely an accident. Thanks for catching, oy vey
```suggestion
  default_api_v3_url: System.get_env("API_V3_URL", "https://api-v3.mbta.com/"),
```

**Pull request merged**